issue_25/get_seq.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Oct 14 11:25:08 2020
+
+@author: Bátora Dániel
+"""
+
+import os 
+from Bio import SeqIO
+from typing import Union
+
+
+
+def get_seq(filepath:str,organism:Union[str, int],temp_dir:str) -> str:
+    
+    """
+    Selects DNA sequence of the desired organims,
```

Comment:
Be clear that what we are extracting is transcript sequences. Also, it might be good to include an example in this docstring as we are expecting a very specific layout of the FASTA ID line (4th "column" and 5th "column"), people who do somehow not have access to the original FASTA file will not be able to use the function easily